doozer/doozerlib/cli/fbc.py

```
@@ -275,6 +275,36 @@ async def run(self):
             if not matching_tags:
                 raise IOError("No matching tags found")
             LOGGER.info("Found %s matching tags: %s", len(matching_tags), matching_tags)
+            # Get all operators with bundles (enabled images with update-csv config)
+            # This matches the logic in olm_bundle.py:list_olm_operators
+            operators_with_bundles = [
+                image.get_component_name()
+                for image in runtime.ordered_image_metas()
+                if image.enabled and image.config.get('update-csv') is not Missing
```

Pylint (4.0.2)

[error] Undefined variable 'Missing'

(E0602)